lib/grape/validations/validators/length_validator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Grape
+  module Validations
+    module Validators
+      class LengthValidator < Base
+        def initialize(attrs, options, required, scope, **opts)
+          @min_length = options[:min]
+          @max_length = options[:max]
+          super
+        end
+
+        def validate_param!(attr_name, params)
+          param = params[attr_name]
+          return unless param.respond_to?(:length)
+          return unless (@min_length && param.length < @min_length) || (@max_length && param.length > @max_length)
```

Comment:
Semantically we probably don't want to treat `@min_length` of zero like this, so use `@min_length.nil?` in the condition?

Reply:
makes sense done 👍